packages/govern-create/deploy/factories.ts

```diff
@@ -0,0 +1,48 @@
+import { HardhatRuntimeEnvironment } from 'hardhat/types'
+import { DeployFunction } from 'hardhat-deploy/types'
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const { deployments, getNamedAccounts } = hre
+  const { deploy } = deployments
+
+  const { deployer } = await getNamedAccounts()
```

Comment:
I agree, but here types are already inferred. What you consider are the benefits of adding type definitions anyways?

Reply:
To have the full type security because the current definition is an implicit any type definition and would provide the full type security you could have with TS.